modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -240,4 +245,9 @@ $ alias coreos-installer='podman run --privileged --pull always --rm \
 ----
 $ coreos-installer iso ignition embed -fi sno/bootstrap-in-place-for-live-iso.ign fcos-live.iso
 ----
-endif::openshift-origin[]
\ No newline at end of file
+endif::openshift-origin[]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://docs.openshift.com/container-platform/{latest}/installing/installing_sno/install-sno-preparing-to-install-sno.html[Requirements for installing OpenShift on a single node].
```

Comment:
This link is not working for me. Getting a 404 error.

Reply:
Thanks for the review! I believe the link won't render until publication, because of the `{latest}` variable.